.github/workflows/bot.yml

```diff
@@ -308,6 +300,49 @@ jobs:
           - scalaProfile: "scala-2.12"
             sparkProfile: "spark3.4"
             sparkModules: "hudi-spark-datasource/hudi-spark3.4.x"
+
+    steps:
+      - uses: actions/checkout@v3
```

Comment:
do you know how did we miss this before?

Reply:
The Github diff makes it harder to read.  I'm adding `test-spark-java11-17-java-tests` and `test-spark-java11-17-scala-tests` jobs only.